packages/elements/src/account/api/index.ts

```diff
@@ -0,0 +1,49 @@
+import ky, { type KyInstance } from 'ky';
+
+import { type UserProfile } from '../types.js';
+
+export type GetAccessToken = () => Promise<string>;
+
+/**
+ * The API client for the Logto account elements
+ *
+ * Used to interact with Account-related backend APIs, including the Profile API.
+ */
+export class LogtoAccountApi {
+  private readonly ky: KyInstance;
```

Comment:
nit: Can we get rid of ky from our elements package?

Reply:
It’s more convenient to implement the functionality using ky. We can evaluate later whether to remove it, but for now, let’s move on.